### ISSUE TITLE
Remove task for publishing single package

### ISSUE THREAD

**Issue by pomek** [ckeditor/ckeditor5-dev#498]
In https://github.com/ckeditor/ckeditor5-dev/pull/431, we split the release process into two steps:

1. bumping version of packages,
2. publishing on npm or/and GitHub.

But the steps above touch the ckeditor5 repository and its packages. 

For releasing a single repository (ckeditor5-react, ckeditor5-angular, ...) we still use a task which bumps a version and publishes everything as a single step. If something bad happens during the task, you cannot call it again because it broke at the middle of the process and before you starting releasing once again, you need to remove everything the task did (we even found a bug – https://github.com/ckeditor/ckeditor5-dev/issues/496 that breaks the process). 

I would like to remove the task for releasing the single package and introducing an option to releasing multi-repositories task which allows handling single package only. 

After introducing proposed changes, we will have the `--dry-run` option everywhere and we will be able to test the release process safe way.

**Comment by pomek** [comment]
The entire logs that we have for ckeditor5 repository, would be awesome in single repositories:

![image](https://user-images.githubusercontent.com/2270764/54117537-4d071580-43f1-11e9-83ec-2dff0843697e.png)

**Comment by Reinmar** [comment]
Hm... I feel like these are two separate processes. I remember that we introduced the other task (single repo release) because it didn't look any like the multi repo one. But you know the code more than I do, so it's your decision to make. Just remember that a de-duplicated code with multiple ifs is usually worse to maintain than two simpler versions of it.

**Comment by pomek** [comment]
Yeah, you're right. When we introduced those tasks, we needed to separate all the things because we didn't know too much about the entire process.

Now, after two years, I can see that we want to do the same things for both versions (single and multi). There is only one difference between those tasks – a number of paths (to repo/repos) where the whole process should be executed. And this difference can be controlled via a parameter (we already support a lof of them).
